html/index.php

```diff
@@ -99,13 +116,29 @@ function getUserReplyText($post_id) {
         <div class="card-body">
           <p class="card-title"><b><?= "{$t['id']}" ?></b> <?= "{$t['name']}" ?> <small><?= "{$t['updated_at']}" ?></small></p>
           <p class="card-text"><?= "{$t['text']}" ?></p>
+          
           <!--返信課題はここから修正しましょう。-->
           <?php if (isset($t['reply_id'])) { ?>
-            <a href="index.php?reply=<?= "{$t['id']}" ?>">[返信する]</a> <a href="/view.php?id=<?= "{$t['reply_id']}" ?>">[返信元のメッセージ]</a>
+            <p><a href="index.php?reply=<?= "{$t['id']}" ?>">[返信する]</a>  <a href="/view.php?id=<?= "{$t['reply_id']}" ?>">[返信元のメッセージ]</a></p>
           <?php } else { ?>
             <p><a href="index.php?reply=<?= "{$t['id']}" ?>">[返信する]</a></p>
           <?php } ?>
           <!--返信課題はここまで修正しましょう。-->
+          <form method="POST" name="like_form">
+            <input type="hidden" name="post_id" value="<?= "{$t['id']}" ?>"/>
+          <!--input type="submit" name="送信"/ -->
+          <?php if (hasFavorite($t['id'],$_SESSION['user_id']) ==0) { ?>
```

Comment:
[must]
`,` の前後の空白や `==` の前後の空白をいれましょう
ここだけコードスタイルが異なり、なぜコードスタイルを替えているのだろうと余計な疑問を持たれてしまうためです。

また、適切に空白を入れたりすることでコードが見やすくなります。
※ 実際には現場、現場でコーディングルールが違うので、それに合わせることになると思います。

```suggestion
          <?php if (hasFavorite($t['id'], $_SESSION['user_id']) == 0) { ?>
```